UIKITDZ/UIKITDZ/Controllers/ViewController.swift

```diff
@@ -0,0 +1,177 @@
+                  !text.isEmpty else {
+                self.welcomeLabel.text = "Привет незнакомец!"
+                self.welcomeLabel.isHidden = false
+                self.additionButton.isHidden = false
+                self.guessTheNumberButton.isHidden = false
+                return
+            }
+            self.welcomeLabel.text = "Привет \(text)!"
+            self.welcomeLabel.isHidden = false
+            self.additionButton.isHidden = false
+            self.guessTheNumberButton.isHidden = false
+        }
+        welcomeAlert.addTextField(configurationHandler: nil)
+        welcomeAlert.addAction(welcomeAlertAction)
+        self.present(welcomeAlert, animated: true, completion: nil)
+    }
+    // алерт стандартный
+    func alert(title: String, message: String, style: UIAlertController.Style, textField: Bool) {
+        let alertController = UIAlertController(title: title, message: message, preferredStyle: style)
+        let alertAction = UIAlertAction(title: "Ok", style: .default) { _ in
+        }
+        alertController.addAction(alertAction)
+        self.present(alertController, animated: true, completion: nil)
+        if textField {
+            alertController.addTextField()
+        }
+    }
+    // алерт кнопки сложения с двумя числами
+    func additionGameAlert() {
+        let additionGameAlert = UIAlertController(title: "Сложение", message: "Введите 2 числа", preferredStyle: .alert)
+        let additionGameAlertAction = UIAlertAction(title: "Ок", style: .default) { _ in
+            guard let numberOne = Int(additionGameAlert.textFields?[0].text ?? ""),
+                  let numberTwo = Int(additionGameAlert.textFields?[1].text ?? "") else {
+                self.alert(title: "Ой!", message: "Вы ввели некорректное значение", style: .alert, textField: false)
+                return
+            }
+            self.alert(title: "Отлично", message: "Cумма равна \(numberOne + numberTwo)",
+                       style: .alert, textField: false)
+        }
+        additionGameAlert.addTextField { texField in
+            texField.placeholder = "Число 1"
+        }
+        additionGameAlert.addTextField { texField in
+            texField.placeholder = "Число 2"
+        }
+        additionGameAlert.addAction(additionGameAlertAction)
+        self.present(additionGameAlert, animated: true, completion: nil)
+    }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(true)
+        welcomeAlert()
+        
+    }
+    // действие при нажатии кнопки сложение
+    @objc private func actionsAdditionButton() {
+        additionGameAlert()
+    }
+    
+    // действие при нататии кнопки угадай число
+    @objc private func guessTheNumberButtonAction() {
+      game(number: nil)
+    }
+    // игра угадай число
+    // функция алертов для игры угадай число
+    func showAlert (title: String, text: String, textfield: Bool) {
```

Comment:
```suggestion
    func showAlert (title: String, text: String, textfield: Bool) {
```
переведи название из комента выше